apps/minaintyg/src/hooks/useLogout.ts

```diff
@@ -1,12 +1,25 @@
 import { useNavigate } from 'react-router-dom'
+import { loginMethodEnum } from '../schema/user.schema'
+import { useGetUserQuery } from '../store/api'
 import { useFakeLogoutMutation } from '../store/testabilityApi'
 
 export function useLogout() {
   const [fakeLogout] = useFakeLogoutMutation()
+  const { data: user } = useGetUserQuery()
   const navigate = useNavigate()
 
   return () => {
-    fakeLogout()
-    navigate('/welcome')
```

Comment:
Should hooks have tests?

Reply:
Sure, I would prefer to test the effect of a hook, but logout is a bit special. 

Added a test for fake logout. SAML logout is a bit trickier to test since the testing environment has no knowledge about browser path's, so there is no /logout path to navigate to. This would be a good candidate cypress tests.